--- a/feeds/base/package/firmware/b43legacy-firmware/Makefile	2016-04-30 15:39:17.000000000 +0000
+++ b/feeds/base/package/firmware/b43legacy-firmware/Makefile	2016-04-30 18:09:24.500755788 +0000
@@ -65,7 +65,7 @@
 	$(INSTALL_DIR) $(1)/lib/firmware
 	b43-fwcutter --unsupported -w $(1)/lib/firmware/ $(DL_DIR)/$(PKG_SOURCE)
   ifneq ($(CONFIG_B43LEGACY_FW_SQUASH),)
-	b43-fwsquash.py "G" "$(CONFIG_B43LEGACY_FW_SQUASH_COREREVS)" "$(1)/lib/firmware/b43legacy"
+	../../../tools/b43-tools/files/b43-fwsquash.py "G" "$(CONFIG_B43LEGACY_FW_SQUASH_COREREVS)" "$(1)/lib/firmware/b43legacy"
   endif
 endef
 
