diff --git a/feeds/base/package/utils/busybox/config/Config.in
b/feeds/base/package/utils/busybox/config/Config.in
index 6a205f6..896aed7 100644
--- a/feeds/base/package/utils/busybox/config/Config.in
+++ b/feeds/base/package/utils/busybox/config/Config.in
@@ -815,29 +815,30 @@ config BUSYBOX_CONFIG_PREFIX

 endmenu

-source package/utils/busybox/config/libbb/Config.in
+source libbb/Config.in

 endmenu

 comment "Applets"

-source package/utils/busybox/config/archival/Config.in
-source package/utils/busybox/config/coreutils/Config.in
-source package/utils/busybox/config/console-tools/Config.in
-source package/utils/busybox/config/debianutils/Config.in
-source package/utils/busybox/config/editors/Config.in
-source package/utils/busybox/config/findutils/Config.in
-source package/utils/busybox/config/init/Config.in
-source package/utils/busybox/config/loginutils/Config.in
-source package/utils/busybox/config/e2fsprogs/Config.in
-source package/utils/busybox/config/modutils/Config.in
-source package/utils/busybox/config/util-linux/Config.in
-source package/utils/busybox/config/miscutils/Config.in
-source package/utils/busybox/config/networking/Config.in
-source package/utils/busybox/config/printutils/Config.in
-source package/utils/busybox/config/mailutils/Config.in
-source package/utils/busybox/config/procps/Config.in
-source package/utils/busybox/config/runit/Config.in
-source package/utils/busybox/config/selinux/Config.in
-source package/utils/busybox/config/shell/Config.in
-source package/utils/busybox/config/sysklogd/Config.in
+source archival/Config.in
+source coreutils/Config.in
+source console-tools/Config.in
+source debianutils/Config.in
+source editors/Config.in
+source findutils/Config.in
+source init/Config.in
+source loginutils/Config.in
+source e2fsprogs/Config.in
+source modutils/Config.in
+source util-linux/Config.in
+source miscutils/Config.in
+source networking/Config.in
+source printutils/Config.in
+source mailutils/Config.in
+source procps/Config.in
+source runit/Config.in
+source selinux/Config.in
+source shell/Config.in
+source sysklogd/Config.in
+
diff --git a/feeds/base/package/utils/busybox/config/networking/Config.in
b/feeds/base/package/utils/busybox/config/networking/Config.in
index 40b3c68..7e5a414 100644
--- a/feeds/base/package/utils/busybox/config/networking/Config.in
+++ b/feeds/base/package/utils/busybox/config/networking/Config.in
@@ -1111,7 +1111,7 @@ config BUSYBOX_CONFIG_FEATURE_TUNCTL_UG
          Allow to specify owner and group of newly created interface.
          340 bytes of pure bloat. Say no here.

-source package/utils/busybox/config/networking/udhcp/Config.in
+source udhcp/Config.in

 config BUSYBOX_CONFIG_IFUPDOWN_UDHCPC_CMD_OPTIONS
        string "ifup udhcpc command line options"
diff --git a/feeds/base/package/utils/busybox/config/util-linux/Config.in
b/feeds/base/package/utils/busybox/config/util-linux/Config.in
index 8c87f53..017b119 100644
--- a/feeds/base/package/utils/busybox/config/util-linux/Config.in
+++ b/feeds/base/package/utils/busybox/config/util-linux/Config.in
@@ -800,6 +800,6 @@ config BUSYBOX_CONFIG_FEATURE_MTAB_SUPPORT
          About the only reason to use this is if you've removed /proc from
          your kernel.

-source package/utils/busybox/config/util-linux/volume_id/Config.in
+source volume_id/Config.in

 endmenu
